assets/diagrams/sequence/figure56.plantuml

```diff
@@ -0,0 +1,158 @@
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006611111A2111139100000
+  00006966666B7666679100000
+  0000266666666666664100000
+  0000000111111111111100000
+  0000000000000000000000000
+}
+
+' declaring skinparam
+skinparam sequenceMessageAlign center
+skinparam shadowing false
+skinparam defaultFontName Verdana
+skinparam monochrome true
+skinparam SequenceLifeLineBackgroundColor WhiteSmoke
+skinparam SequenceLifeLineBorderColor Black
+skinparam ActorFontStyle Bold
+skinparam ActorFontSize 20
+skinparam ParticipantFontStyle Bold
+skinparam ParticipantFontSize 20
+skinparam ParticipantBackgroundColor WhiteSmoke
+skinparam ArrowColor Black
+
+skinparam actor {
+  Bordercolor none
+  Backgroundcolor none
+  shadowing false
+}
+
+skinparam participant {
+  shadowing true
+}
+
+hide footbox
+
+' declare title
+' title Commit notification where commit of transfer in Switch failed
+
+' Actor Keys:
+' participant - FSP(Payer/Payee) and Switch
+' actor - Payee/Payer
+
+' declare actors
+actor "<$actor>\nPayer" as Payer
+participant "Payer\nFSP" as PayerFSP
+participant "Switch" as Switch
+participant "Payee\nFSP" as PayeeFSP
+actor "<$actor>\nPayee" as Payee
+
+' start flow
+Payer ->> PayerFSP: I would like to pay 100 USD\nto +123456789
+activate PayerFSP
+PayerFSP -> PayerFSP: Lookup +123456789\n(process not shown here)
+PayerFSP -[hidden]> Switch
+deactivate PayerFSP
+PayerFSP -[hidden]> Switch
+activate PayerFSP
+PayerFSP -> PayerFSP: Quote (process\nnot shown here)
+PayerFSP -[hidden]> Switch
+deactivate PayerFSP
+PayerFSP -[hidden]> Switch
+activate PayerFSP
+PayerFSP -> PayerFSP: Reserve transfer from Payer\naccount to Switch account
+PayerFSP -[hidden]> Switch
+deactivate PayerFSP
+PayerFSP -[hidden]> Switch
+activate PayerFSP
+PayerFSP ->> Switch: **POST /transfers**\n(Transfer ID, condition, ILP Packet\nincluding transaction details,\nexpiry=30 seconds)
+activate Switch
+PayerFSP <<-- Switch: **HTTP 202** (Accepted)
+Switch -> Switch: Reserve transfer from\nPayer FSP to Payee FSP
+Switch ->> PayeeFSP: **POST /transfers**\n(Transfer ID, condition, ILP Packet\nincluding transaction details,\nexpiry=20 seconds)
+activate PayeeFSP
+Switch <<-- PayeeFSP: **HTTP 202** (Accepted)
+PayeeFSP -> PayeeFSP: Perform (or Reserve) transfer from Switch\naccount to Payee account,\ngenerate fulfilment
+PayeeFSP -> PayeeFSP: (In case of reserved) Request commit notification\nfrom Switch by sending transfer\nstate as reserved
+Switch <<- PayeeFSP: **PUT /transfers/**<i><ID></i>\n(Fulfilment,\ntransferState in [RESERVED, COMMITTED])
+Switch -->> PayeeFSP: **HTTP 200** (OK)
+Switch -> Switch: Validation of **PUT /transfers/**<i><ID></i> failed and so\ncommit transfer from\nPayer FSP to Payee FSP\nfailed, notify both FSPs
+Switch ->> PayeeFSP: **PATCH /transfers/**<i><ID></i>**/error**
```

Comment:
The **PATCH /transfers/**_{ID}_**/error** notification must not be sent in version 1.2 when `transferState` `RESERVED` is used, see https://github.com/mojaloop/mojaloop-specification/issues/103#issuecomment-970407712.